guides/common/modules/proc_configuring-dhcp-native-ms.adoc

```diff
@@ -0,0 +1,16 @@
+[id="Configuring_dhcp_native_ms_{context}"]
+= Configuring dhcp_native_ms
+
+The *dhcp_native_ms* plugin manages IP reservations in Microsoft Active Directory using its native API.
```

Comment:
Perhaps it's not clear from this, but this must run on Windows. Given we only document Linux I wonder if we should document this.

In an ideal world we would have a guide "Configuring Smart Proxy on Windows" which would document how to run this and dns_dnscmd. Until then I'm leaning to dropping it.